NorthstarDLL/masterserver.cpp

```diff
@@ -231,9 +235,10 @@ void MasterServerManager::RequestServerList()
 					if (!serverObj.HasMember("id") || !serverObj["id"].IsString() || !serverObj.HasMember("name") ||
 						!serverObj["name"].IsString() || !serverObj.HasMember("description") || !serverObj["description"].IsString() ||
 						!serverObj.HasMember("map") || !serverObj["map"].IsString() || !serverObj.HasMember("playlist") ||
-						!serverObj["playlist"].IsString() || !serverObj.HasMember("playerCount") || !serverObj["playerCount"].IsNumber() ||
-						!serverObj.HasMember("maxPlayers") || !serverObj["maxPlayers"].IsNumber() || !serverObj.HasMember("hasPassword") ||
-						!serverObj["hasPassword"].IsBool() || !serverObj.HasMember("modInfo") || !serverObj["modInfo"].HasMember("Mods") ||
+						!serverObj["playlist"].IsString() || !serverObj.HasMember("region") || !serverObj["region"].IsString() ||
+						!serverObj.HasMember("playerCount") || !serverObj["playerCount"].IsNumber() || !serverObj.HasMember("maxPlayers") ||
+						!serverObj["maxPlayers"].IsNumber() || !serverObj.HasMember("hasPassword") || !serverObj["hasPassword"].IsBool() ||
+						!serverObj.HasMember("modInfo") || !serverObj["modInfo"].HasMember("Mods") ||
```

Comment:
Uh, someone double check that I didn't remove some vital check here :P

Reply:
I hate this if statement so much

Reply:
Looks good to me, though